### PR TITLE
BinaryNinja.Demo version idk

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98934)

### CODE REVIEW

File: manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
+PackageLocale: en-US
+Publisher: Binary Ninja
+PackageName: Binary Ninja Demo
+License: Proprietary
+ShortDescription: Binary Ninja
```
Comment (Trenly) [inline manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml:10]: Provide a better description of what Binary Ninja is used for

File: manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
+PackageLocale: en-US
+Publisher: Binary Ninja
```
Comment (Trenly) [inline manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml:7]:
```suggestion
Publisher: Vectore 35, Inc.
```
Must match control panel

File: manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
+PackageLocale: en-US
+Publisher: Binary Ninja
+PackageName: Binary Ninja Demo
```
Comment (Trenly) [inline manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml:8]:
```suggestion
PackageName: Binary Ninja
```

File: manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
```
Comment (Trenly) [inline manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.yaml:5]:
```suggestion
PackageVersion: 1.0
```
When package version is null, always use some form of semantic version

File: manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
```
Comment (Trenly) [inline manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.installer.yaml:5]:
```suggestion
PackageVersion: 1.0
```
When package version is null, always use some form of semantic version

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Validation is failing because the locale file has version `1.0` but the other 2 have `idk`: 
>  Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 1.0 File: BinaryNinja.Demo.locale.en-US.yaml

While using a semantic version is a best practice, just having them match will pass validation.

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**